lib/membrane_ffmpeg_video_filter/text_overlay.ex

```diff
@@ -130,6 +130,24 @@ defmodule Membrane.FFmpeg.VideoFilter.TextOverlay do
   end
 
   @impl true
+  def handle_process(
+        :input,
+        %Buffer{pts: nil} = buffer,
+        _ctx,
+        %{text_intervals: intervals} = state
+      ) do
+    case intervals do
+      [{{0, :infinity}, _text}] ->
+        buffer = Native.apply_filter!(buffer, state.native_state)
+        {{:ok, [buffer: {:output, buffer}]}, state}
```

Comment:
Just a minor styling issue - you can use Elixir syntax sugar for keyword list at the end
```suggestion
        {{:ok, buffer: {:output, buffer}}, state}
```